Problem Building TA through TensorWrapper

Using master I run into the problem:

```.txt
Cloning into 'linalg-cmake-modules-src'...
[cmake] fatal: reference is not a tree: 916be6da6702eb2d5ed1f921764256d75dc78dfd
[cmake] CMake Error at linalg-cmake-modules-subbuild/linalg-cmake-modules-populate-prefix/tmp/linalg-cmake-modules-populate-gitclone.cmake:40 (message):
[cmake]   Failed to checkout tag: '916be6da6702eb2d5ed1f921764256d75dc78dfd'
```

Can anyone else verify on a clean build?

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -40,7 +40,7 @@ cpp_find_or_build_dependency(
 cpp_find_or_build_dependency(
     tiledarray
     URL github.com/ValeevGroup/tiledarray
-    VERSION 6786a55ea4490e88877511dd952e45fe0d2ac069
+    VERSION 36e2ad205c21c339434dd0ef8f4f1467e7e26037
     BUILD_TARGET tiledarray
     FIND_TARGET tiledarray
     CMAKE_ARGS TA_BUILD_UNITTEST=OFF

